src/main/java/uk/gov/hmcts/probate/controller/ExceptionRecordController.java

```diff
@@ -120,9 +122,56 @@
         return ResponseEntity.ok(callbackResponse);
     }
 
-    private void logCallback(SuccessfulTransformationResponse callbackResponse) {
+    @ApiOperation(value = "Updates a case based on availability of OCR data and documents", notes = "Will return errors if unsuccessful or no new documents found.")
+    @ApiResponses({
+            @ApiResponse(code = 200, response = ValidationResponse.class, message = "Validation executed successfully"),
+            @ApiResponse(code = 400, message = "Request failed due to malformed syntax"),
+            @ApiResponse(code = 403, message = "S2S token is not authorized, missing or invalid"),
+            @ApiResponse(code = 404, message = "Form type not found")
+    })
+    @PostMapping(path = "/update-case",
```

Comment:
This controller method needs a unit test.

Reply:
added tests